libs/coin-modules/coin-sui/src/logic/lastBlock.ts

```diff
@@ -0,0 +1,11 @@
+import { BlockInfo } from "@ledgerhq/coin-framework/lib/api/types";
+import { getLastBlock } from "../network/sdk";
+
+export async function lastBlock(): Promise<BlockInfo> {
+  const { digest, sequenceNumber } = await getLastBlock();
+
+  return {
+    height: BigInt(sequenceNumber) as unknown as number,
```

Comment:
```suggestion
    height: Number(sequenceNumber),
```

Reply:
we had a discussion with in slack, to prevent overflow we can expect BigInt type, changed for now